app/scripts/modules/core/src/application/config/trafficGuard/trafficGuardConfig.component.ts

```diff
@@ -23,8 +23,8 @@ export interface ITrafficGuard {
 export class TrafficGuardConfigController {
 
   public application: Application;
-  public accounts: IAccountDetails[] = [];
```

Comment:
Any reason to drop this initialization? (It looks like everything will be OK without it, but who knows)